<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Connections:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableConnections">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnNewConn">
         <property name="text">
          <string>New Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveBus">
         <property name="text">
          <string>Remove Selected Connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btnActivateAll">
         <property name="text">
          <string>Activate All Connections</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeactivateAll">
         <property name="text">
          <string>Deactivate All Connections</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lConnectionType">
       <property name="text">
        <string>Connection Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item>
      <widget class="QLabel" name="lPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>341</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckSingleWire">
       <property name="text">
        <string>Single Wire Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckListenOnly">
       <property name="text">
        <string>Listen Only Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckEnabled">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>CAN Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSpeed">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnOK">
         <property name="text">
          <string>Create New Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRevert">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="ckEnableConsole">
       <property name="text">
        <string>Enable Console</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device Console:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textConsole">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Send:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="btnSendHex">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send Hex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSendText">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClearDebug">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
